src/amberscript/parser_buffer_test.cc

```diff
@@ -48,6 +48,10 @@ class DummyDelegate : public amber::Delegate {
 
     return {};
   }
+
+  amber::Result LoadFile(const std::string, std::vector<char>&) const override {
+    return Result("DummyDelegate::LoadFile not implemented");
```

Comment:
```suggestion
    return Result("PlaceholderDelegate::LoadFile not implemented");
```

Reply:
Are you sure? The class name is DummyDelegate.

Reply:
Oh, you're right. I'd thought I'd renamed that already. Leave this and I'll rename the delegate in the future.